### DISCUSSION TITLE
xmlns:xsi attribute

### DISCUSSION THREAD

**Discussion by khaiyihong** [prowide/prowide-iso20022#69] [Q&A]
Hi, when I generate the message for PAIN 008.001.02 using MxPain00800102, the output is

`<Document xmlns="urn:iso:std:iso:20022:tech:xsd:pain.008.001.02">`

May I know if I can add the **xmlns:xsi** attribute to `<Document>`, like the following? 

`<Document xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="urn:iso:std:iso:20022:tech:xsd:pain.008.001.02">`

My code is as following
```
MxWriteConfiguration config = new MxWriteConfiguration();
config.documentPrefix = null;
config.headerPrefix = null;

System.out.println(mx.message(config));
```
I am using version SRU2022-9.3.3.

```
<dependency>
    <groupId>com.prowidesoftware</groupId>
    <artifactId>pw-iso20022</artifactId>
    <version>SRU2022-9.3.3</version>
 </dependency>
```

**Comment by ptorres-prowide** [comment]
Please refer to the documentation for instructions on how to do this:
[ISO 20022 Build Documentation](https://dev.prowidesoftware.com/latest/open-source/iso20022/iso20022-build/)

Additionally, you can check these tests for code examples:
[MxWriteTest.java](https://github.com/prowide/prowide-iso20022/blob/main/iso20022-core/src/test/java/com/prowidesoftware/swift/model/mx/MxWriteTest.java)
